### PR TITLE
Add migrate task to Laravel recipe

### PR DESCRIPTION
**Pull request by lucasmichot**
Add missing migrate task to deployment

### CODE REVIEW

File: recipe/laravel.php [hunk 1]
```diff
@@ -17,6 +17,13 @@
 set('writable_dirs', ['storage', 'vendor']);
 
 /**
+ * Migrate database
+ */
+task('deploy:migrate', function () {
+    run("cd {{release_path}} && ./artisan migrate --force");
```
Comment (GrahamCampbell) [inline recipe/laravel.php]: :-1: for forcing

Comment (GrahamCampbell) [inline recipe/laravel.php]: Does laravel support running with "no interaction". If not, we need to do something about that. :)

Comment (lucasmichot) [inline recipe/laravel.php]: True

**Comment by mbardelmeijer** [comment]
:+1:

File: recipe/laravel.php (hunk 1 above)
Comment (elfet) [inline recipe/laravel.php]: Maybe not forcing?

**Pull request closed without merging**